src/connector/src/sink/encoder/json.rs

```diff
@@ -402,7 +406,7 @@ pub(crate) fn schema_type_mapping(rw_type: &DataType) -> &'static str {
         DataType::List(_) => "array",
         DataType::Bytea => "bytes",
         DataType::Jsonb => "string",
-        DataType::Serial => "int32",
+        DataType::Serial => "string",
```

Comment:
I suppose this branch is unreachable previously..?

Reply:
I think so...